generators/client/templates/angular/src/main/webapp/app/admin/configuration/_configuration.component.html

```diff
@@ -3,12 +3,12 @@ <h2 jhi-translate="configuration.title">Configuration</h2>
 
     <span jhi-translate="configuration.filter">Filter (by prefix)</span> <input type="text" [(ngModel)]="filter" class="form-control">
     <label>Spring configuration</label>
-    <table class="table table-condensed table-striped table-bordered table-responsive"
+    <table class="table table-sm table-striped table-bordered"
```

Comment:
`Responsive tables no longer require a wrapping element. Instead, just put the .table-responsive right on the <table>.`

to be exact